Classes/Private/Manager/Objects/OCTMessageCall.m

```diff
@@ -22,7 +23,28 @@ - (NSString *)description
 {
     NSString *description = [super description];
 
-    return [description stringByAppendingFormat:@"\ncallDuration %f", self.callDuration];
+    return [description stringByAppendingString:[self typeDescription]];
+}
+
+#pragma mark - Private
+
+- (NSString *)typeDescription
+{
+    NSString *description;
+    switch (self.callType) {
+        case OCTMessageCallTypeDial:
+            description = @"Call dial";
+            break;
+        case OCTMessageCallTypeEnd:
+            description = [[NSString alloc] initWithFormat:@"Call ended %f seconds", self.callDuration];
+            break;
+        case OCTMessageCallTypeMissed:
+            description = @"Call missed";
+            break;
+        default:
+            break;
```

Comment:
There is no need in `default` statement in case if you implement all options from enum. In case if one more option would be added - you'll get compile error and won't forget to add it here.